src/Xamarin.Android.Build.Tasks/packages.config

```diff
@@ -7,6 +7,8 @@
   <package id="Microsoft.Build.Framework" version="14.3.0" targetFramework="net472" />
   <package id="Microsoft.Build.Tasks.Core" version="14.3.0" targetFramework="net472" />
   <package id="Microsoft.Build.Utilities.Core" version="14.3.0" targetFramework="net472" />
+  <package id="Microsoft.DotNet.ApiCompat" version="5.0.0-beta.19602.1" targetFramework="net472" />
```

Comment:
Why add these to `src/Xamarin.Android.Build.Tasks` when it's `build-tools/Xamarin.Android.Tools.BootstrapTasks` which actually uses these utilities?

Reply:
I could not do it before because of a bug on the build prepare. Fixed by https://github.com/xamarin/xamarin-android/pull/3997

Moved the packages to Xamarin.Android.Tools.BootstrapTasks